keyboards/darkproject/kd83a_bfg_edition/kd83a_bfg_edition.c

```diff
@@ -0,0 +1,271 @@
+    {0, CS7_SW9,   CS8_SW9,   CS9_SW9},   //  38, k82, I
+    {0, CS7_SW10,  CS8_SW10,  CS9_SW10},  //  39, k92, O
+    {0, CS7_SW11,  CS8_SW11,  CS9_SW11},  //  40, ka2, P
+    {0, CS7_SW12,  CS8_SW12,  CS9_SW12},  //  41, kb2, [
+    {1, CS1_SW8,   CS2_SW8,   CS3_SW8},   //  42, kc2, ]
+    {1, CS1_SW9,   CS2_SW9,   CS3_SW9},   //  43, kd3, "\\"
+    {1, CS4_SW3,   CS5_SW3,   CS6_SW3},   //  44, k06, Del
+  
+    {0, CS10_SW1,  CS11_SW1,  CS12_SW1},  //  45, k03, Caps Lock
+    {0, CS10_SW2,  CS11_SW2,  CS12_SW2},  //  46, k13, A
+    {0, CS10_SW3,  CS11_SW3,  CS12_SW3},  //  47, k23, S
+    {0, CS10_SW4,  CS11_SW4,  CS12_SW4},  //  48, k33, D
+    {0, CS10_SW5,  CS11_SW5,  CS12_SW5},  //  49, k43, F
+    {0, CS10_SW6,  CS11_SW6,  CS12_SW6},  //  50, k53, G
+    {0, CS10_SW7,  CS11_SW7,  CS12_SW7},  //  51, k63, H
+    {0, CS10_SW8,  CS11_SW8,  CS12_SW8},  //  52, k73, J
+    {0, CS10_SW9,  CS11_SW9,  CS12_SW9},  //  53, k83, K
+    {0, CS10_SW10, CS11_SW10, CS12_SW10}, //  54, k93, L
+    {0, CS10_SW11, CS11_SW11, CS12_SW11}, //  55, ka3, ;
+    {0, CS10_SW12, CS11_SW12, CS12_SW12}, //  56, kb3, '
+    {1, CS1_SW11,  CS2_SW11,  CS3_SW11},  //  57, kc3, Enter
+    {1, CS4_SW4,   CS5_SW4,   CS6_SW4},   //  58, k26, PgUp
+   
+    {0, CS13_SW1,  CS14_SW1,  CS15_SW1},  //  59, k04, Shift_L
+    {0, CS13_SW2,  CS14_SW2,  CS15_SW2},  //  60, k24, Z
+    {0, CS13_SW3,  CS14_SW3,  CS15_SW3},  //  61, k34, X
+    {0, CS13_SW4,  CS14_SW4,  CS15_SW4},  //  62, k44, C
+    {0, CS13_SW5,  CS14_SW5,  CS15_SW5},  //  63, k54, V
+    {0, CS13_SW6,  CS14_SW6,  CS15_SW6},  //  64, k64, B
+    {0, CS13_SW7,  CS14_SW7,  CS15_SW7},  //  65, k74, N
+    {0, CS13_SW8,  CS14_SW8,  CS15_SW8},  //  66, k84, M
+    {0, CS13_SW9,  CS14_SW9,  CS15_SW9},  //  67, k94, ,
+    {0, CS13_SW10, CS14_SW10, CS15_SW10}, //  68, ka4, .
+    {0, CS13_SW11, CS14_SW11, CS15_SW11}, //  69, kb4, /
+    {1, CS4_SW8,   CS5_SW8,   CS6_SW8},   //  70, kd4, Shift_R
+    {1, CS4_SW9,   CS5_SW9,   CS6_SW9},   //  71, k17, Up
+    {1, CS4_SW7,   CS5_SW7,   CS6_SW7},   //  72, k36, PgDn
+
+    {0, CS16_SW1,  CS17_SW1,  CS18_SW1},  //  73, k05, Ctrl_L
+    {0, CS16_SW2,  CS17_SW2,  CS18_SW2},  //  74, k15, Win_L
+    {0, CS16_SW3,  CS17_SW3,  CS18_SW3},  //  75, k25, Alt_L
+    {0, CS16_SW6,  CS17_SW6,  CS18_SW6},  //  76, k65, Space
+    {0, CS16_SW9,  CS17_SW9,  CS18_SW9},  //  77, k95, Alt_R
+    {0, CS16_SW10, CS17_SW10, CS18_SW10}, //  78, ka5, FN
+    {0, CS16_SW12, CS17_SW12, CS18_SW12}, //  79, kc5, Ctrl_R
+    {1, CS4_SW10,  CS5_SW10,  CS6_SW10},  //  80, k07, Left
+    {1, CS4_SW11,  CS5_SW11,  CS6_SW11},  //  81, k27, Down
+    {1, CS10_SW5,  CS11_SW5,  CS12_SW5},  //  82, k37, Right
+   
+    {1, CS10_SW10, CS11_SW10, CS12_SW10}, //  83, Z1
+    {1, CS10_SW11, CS11_SW11, CS12_SW11}, //  84, Z2  
+};
+
+#undef ___
+#define ___ NO_LED
+#define RGB_LAYOUT LAYOUT
+
+led_config_t g_led_config = {
+    RGB_LAYOUT(  0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 
```

Comment:
This now throws lint warnings and should be reworked to nor use the LAYOUT macros

Reply:
> This now throws lint warnings and should be reworked to not use the LAYOUT macros

Made changes according to the request. Could you please review it.

Reply:
@zvecr

Reply:
Still uses `RGB_LAYOUT` so still no good

Reply:
@zvecr Changes are made. Could you please review it.

Reply:
@zvecr Changes made accordingly in kd83a_bfg_edition.c file. Could you please review it.

Reply:
@zvecr Could you please help me to review the changes.
Thank you